test/Faker/Core/BloodTest.php

```diff
@@ -10,16 +10,44 @@ final class BloodTest extends TestCase
 {
     public function testBloodType(): void
     {
-        self::assertContains($this->faker->bloodType(), ['A', 'AB', 'B', 'O']);
+        $resultSet = [];
+
+        for ($i = 0; $i < 17; ++$i) {
```

Comment:
@anubisthejackle 

Perhaps it makes sense to use the `UniqueGenerator` in the test to exhaust all values?

```php
public function testBloodType(): void
{
    $expected = [
        'A',
        'AB',
        'B',
        'O',
    ];

    $faker = new UniqueGenerator($this->faker);

    for ($i = 0; $i < count($expected); ++$i) {
        $generated[] = $faker->bloodType();
    }

    self::assertEqualsCanonicalizing($expected, $generated);
}
```

What do you think?

Reply:
Yes, that's exactly what I was looking for. I'll update the test.